odoo/addons/test_assetsbundle/tests/test_assetsbundle.py

```diff
@@ -34,18 +34,21 @@ def setUp(self):
         super(TestJavascriptAssetsBundle, self).setUp()
         self.jsbundle_xmlid = 'test_assetsbundle.bundle1'
         self.cssbundle_xmlid = 'test_assetsbundle.bundle2'
+        self.env['res.lang'].load_lang('ar_SY')
 
     def _get_asset(self, xmlid, env=None):
         env = (env or self.env)
         files, remains = env['ir.qweb']._get_asset_content(xmlid, env.context)
         return AssetsBundle(xmlid, files, env=env)
 
-    def _any_ira_for_bundle(self, type):
+    def _any_ira_for_bundle(self, type, env=None):
```

Comment:
@msh-odoo You made this because `with_context` does not work to call the test right?
If this is only to change the language context, use something minimal like the `lang=` or `context=`